src/math/rpy.hpp

```diff
@@ -42,12 +36,8 @@ namespace pinocchio
     /// around axis \f$\alpha\f$.
     ///
     template<typename Vector3Like>
-    Eigen::Matrix<typename Vector3Like::Scalar,3,3,PINOCCHIO_EIGEN_PLAIN_TYPE(Vector3Like)::Options>
-    rpyToMatrix(const Eigen::MatrixBase<Vector3Like> & rpy)
-    {
-      PINOCCHIO_ASSERT_MATRIX_SPECIFIC_SIZE(Vector3Like, rpy, 3, 1);
-      return rpyToMatrix(rpy[0], rpy[1], rpy[2]);
-    }
+    inline Eigen::Matrix<typename Vector3Like::Scalar,3,3,PINOCCHIO_EIGEN_PLAIN_TYPE(Vector3Like)::Options>
```

Comment:
Oh, hadn't thought about it. Well, _all_ of Pinocchio's functions are template functions, do I guess it can be removed everywhere :) Anyhow, I'll remove it here

Reply:
Yes, you're right. I plan to remove them in the future big release (aka Pinocchio 3).